diff --git a/openvdb/openvdb/python/CMakeLists.txt b/openvdb/openvdb/python/CMakeLists.txt
index 82d7c5ac..cc810cbb 100644
--- a/openvdb/openvdb/python/CMakeLists.txt
+++ b/openvdb/openvdb/python/CMakeLists.txt
@@ -253,9 +253,9 @@ endif()
 add_library(pyopenvdb SHARED ${OPENVDB_PYTHON_MODULE_SOURCE_FILES})
 set_target_properties(pyopenvdb PROPERTIES PREFIX "")  # no 'lib' prefix
 if(UNIX)
-  set_target_properties(pyopenvdb PROPERTIES SUFFIX ".so") # must be .so (not .dylib)
+  set_target_properties(pyopenvdb PROPERTIES SUFFIX ".${Python_SOABI}.so") # must be .so (not .dylib)
 elseif(WIN32)
-  set_target_properties(pyopenvdb PROPERTIES SUFFIX ".pyd") # .pyd on windows
+  set_target_properties(pyopenvdb PROPERTIES SUFFIX ".${Python_SOABI}.pyd") # .pyd on windows
 endif()
 
 target_link_libraries(pyopenvdb PUBLIC
